sansa-ml-spark/src/main/scala/net/sansa_stack/ml/spark/entity_resolution/predicatesfilteration.txt

```diff
@@ -0,0 +1,13 @@
+owl:sameas
```

Comment:
This file is quite static and placed in the wrong place. Usually, files should be placed into resources and not as `.txt` extension. Please, consider using `application.config` where you can directly place them into a list instead of lines.

Reply:
Done.